Onboarding/QA/tests/profile-pregnancy-health_test.exs

```diff
@@ -1323,6 +1323,29 @@ defmodule ProfilePregnancyHealthTest do
       })
     end
 
+    test "edd day then above max day error february 29" do
+      months = get_months()
+      month_words = get_month_words(months)
+      {list_of_months, _edd_confirmation_text, _full_edd} = get_edd(months, month_words)
+      month = elem(Enum.at(list_of_months, 0), 0)
+
+      setup_flow()
+      |> FlowTester.set_fake_time(~U[2024-02-29 00:00:00Z])
+      |> Helpers.init_contact_fields()
+      |> init_contact_fields()
+      |> init_pregnancy_info()
+      |> FlowTester.start()
+      |> receive_message(%{})
+      |> FlowTester.send(button_label: "I'm pregnant")
+      |> receive_message(%{})
+      |> FlowTester.send(month)
+      |> receive_message(%{})
+      |> FlowTester.send("30")
+      |> receive_message(%{
+        text: "I don't understand your reply.\r\n\r\n👇🏽  Please try that again and respond with the number that comes before your answer."
+      })
+    end
+
     # TODO: Tests for Feb, long months, and short months. This requires us to be able to mock the return value of now() so that we can dictacte what options
     # are available in the list of months.
```

Comment:
Is this comment still relevant?

Reply:
I've added some more tests for long and short months and removed the comment